docs/src/main/asciidoc/update-to-quarkus-3.adoc

```diff
@@ -0,0 +1,55 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="update-projects-to-quarkus-3-automatically-howto"]
+= Update projects to Quarkus 3 automatically
+include::_attributes.adoc[]
+:categories: core
+:extension-status: "experimental"
+:summary: Update projects from Quarkus 2.13 and later to Quarkus 3.
+
+include::{includes}/extension-status.adoc[]
+
+As a developer, you can update your projects from Quarkus 2.13 and later to Quarkus 3 by running an update command.
+
+The update command uses OpenRewrite recipes to automatically update most of your project's dependencies, source code, and documentation. These recipes cover many but not all of the items described in the link:https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0[Migration Guide 3.0].
```

Comment:
```suggestion
The update command primarily uses OpenRewrite recipes to automate updating your project's dependencies, source code, and documentation. These recipes cover many but not all of the items described in the link:https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0[Migration Guide 3.0].
```

not sure about this one but isn't the connotation of using "automatically"  that it will all be done for me; where "automate" is more a "make it easier for me" which seems like a better signal to send.

Reply:
Thanks. Updated. Good point.